website/client/src/components/header/menu.vue

```diff
@@ -11,15 +11,16 @@
       type="dark"
       :class="navbarZIndexClass"
     >
-      <b-navbar-brand
```

Comment:
@projectormato sorry I had missed the existing label here, could you revert your last commit?